bin/aws-s3-creator.ts

```diff
@@ -1,21 +1,57 @@
 #!/usr/bin/env node
 import 'source-map-support/register';
+
 import * as cdk from 'aws-cdk-lib';
+import * as dotenv from 'dotenv';
+import { Aspects } from 'aws-cdk-lib';
+import { ApplyTags } from '../utils/apply-tag';
+import { AwsSolutionsChecks } from 'cdk-nag';
+import { checkEnvVariables } from '../utils/check-environment-variable';
 import { AwsS3CreatorStack } from '../lib/aws-s3-creator-stack';
+import { AwsS3CreatorStackProps } from '../lib/AwsS3CreatorStackProps';
 
+dotenv.config(); // Load environment variables from .env file
 const app = new cdk.App();
-new AwsS3CreatorStack(app, 'AwsS3CreatorStack', {
-  /* If you don't specify 'env', this stack will be environment-agnostic.
-   * Account/Region-dependent features and context lookups will not work,
-   * but a single synthesized template can be deployed anywhere. */
+const appAspects = Aspects.of(app);
+
+// check environment variables
+checkEnvVariables('APP_NAME', 'OWNER', 'CDK_DEPLOY_REGION', 'ENVIRONMENT', 'S3_BUCKET_NAME');
+
+const { CDK_DEFAULT_ACCOUNT: account, CDK_DEFAULT_REGION: region } = process.env;
+
+const cdkRegion = process.env.CDK_DEPLOY_REGION!;
+const deployEnvironment = process.env.ENVIRONMENT!;
+
+
+const appName = process.env.APP_NAME!;
+const owner = process.env.OWNER!;
+const s3BucketName = process.env.S3_BUCKET_NAME!;
```

Comment:
**Suggestion:** The `process.env` variables are being accessed directly with non-null assertion (`!`). This can lead to runtime errors if the environment variables are not set. Instead, provide a default value or handle the case where the variable might be undefined.
> 🌅  **High Priority** 
 > [category: **`possible issue`**, importance: **`8`**]
> **Reason:** Providing default values for environment variables is a crucial improvement for robustness and prevents runtime errors if any variable is unset.
 

```suggestion
const cdkRegion = process.env.CDK_DEPLOY_REGION || 'default-region';
const deployEnvironment = process.env.ENVIRONMENT || 'default-environment';
const appName = process.env.APP_NAME || 'default-app-name';
const owner = process.env.OWNER || 'default-owner';
const s3BucketName = process.env.S3_BUCKET_NAME || 'default-bucket-name';
```